src/components/UNSTABLE__LayoutModules/ListItemModule/ListItemModule.stories.js

```diff
@@ -0,0 +1,239 @@
+/**
+ * @file List item module stories.
+ * @copyright IBM Security 2021
+ */
+
+import {
+  Bee16,
+  InProgress16,
+  Locked16,
+  UserAvatar20,
+} from '@carbon/icons-react';
+import React from 'react';
+
+import { getDocsParameters } from '../../../../.storybook';
+import withResponsive from '../../../../.storybook/decorators';
+
+import { ICA, ListItemModule, ProfileImage, StatusIcon, Tag } from '../../..';
+
+import getTitle from '../stories';
+import page from './index.mdx';
+
+export default {
+  title: getTitle(ListItemModule),
+  component: ListItemModule,
+  parameters: {
+    docs: { page },
+
+    ...getDocsParameters(),
+  },
+  decorators: [withResponsive],
+};
+
+export const Default = () => (
+  <ListItemModule href="#">
+    {({ Column, getLayoutProps }) => (
+      <>
+        <Column>
+          <h2 {...getLayoutProps({ type: 'title' })}>List title</h2>
+
+          <span {...getLayoutProps({ type: 'label' })}>Label</span>
+        </Column>
+      </>
```

Comment:
There's no need for wrapping a single child in a Fragment

Reply:
Thanks for catching these!